packages/compiler/server/serverlib.ts

```diff
@@ -613,25 +613,26 @@ export function createServer(host: ServerHost): Server {
     }
   }
 
-  async function getDocument(type: Type): Promise<string> {
+  async function getTypeDetails(type: Type): Promise<string> {
     return type.kind;
```

Comment:
Starting to look good. I think we can improve a little to have more information here than just the kind.

This function is going to return markdown so lets try to represent the following nicely:
- Kind 
- Fully qualified name
- If there is any `getDoc` with its value but make sure to specify this is the api doc as we'll have actual `//` documentation as well later.